source/containers.tex

```diff
@@ -10642,16 +10642,16 @@
     index_type size_;           // \expos
   };
 
```

Comment:
[The similar change to `string_view` adds a reference to the deduction guide](https://github.com/cplusplus/draft/pull/3454/files#diff-dce54a16ad891d254c611a40d1a8564aR4112-R4114). Their stable names also differ. I suggest coordinating it.

`function`'s deduction guide is under "constructors and destructors". A simple search of "deduct" doesn't show any other deduction guide with detailed specifications. This is to consider which style to follow (deduction guides with their own subclause or being along constructors).

Reply:
In this case, we already have more than a page of constructors etc, so it seems eminently reasonable to invest a new subclause for the two deduction guides. (I'm also willing to consider the same for other subclauses with a similar footprint.)